cosmos/airflow/graph.py

```diff
@@ -371,32 +370,6 @@ def generate_task_or_group(
     return task_or_group
 
 
-def _add_dbt_compile_task(
```

Comment:
As we agreed, I've added a task to follow up on this:
https://github.com/astronomer/astronomer-cosmos/issues/1477

Ideally, we'd compare both approaches before 1.9, but I'm happy with us to go ahead with approach of #1474 in 1.9 - it is already a significant improvement when compared to 1.7.0, since it solves the existing SQL query issue.

Reply:
I agree, thanks for creating the follow-up task.